base/src/main/java/io/spine/validate/option/Min.java

```diff
@@ -21,32 +21,29 @@
 package io.spine.validate.option;
 
 import com.google.errorprone.annotations.Immutable;
-import com.google.errorprone.annotations.ImmutableTypeParameter;
+import io.spine.code.proto.FieldContext;
 import io.spine.option.MinOption;
 import io.spine.option.OptionsProto;
-import io.spine.validate.FieldValue;
+import io.spine.validate.Constraint;
 
 /**
  * An option that defines a minimum value for a numeric field.
- *
- * @param <V>
- *         numeric value type that this option is applied to
  */
 @Immutable
-final class Min<@ImmutableTypeParameter V extends Number & Comparable<V>>
-        extends FieldValidatingOption<MinOption, V> {
+final class Min
```

Comment:
Since we no longer having lengthy generic params, consider having the class declaration on the same line (like in `Max`).

Reply:
Please also consider such a layout for declarations of the classes derived from `RangedConstraint`.